vendor/engines/c2po/app/views/c2po/facility_accounts/show/_additional_account_fields.html.haml

```diff
@@ -0,0 +1,5 @@
+- if @account.is_a?(CreditCardAccount)
```

Comment:
I think that this should be `account` instead. (since we're passing it this way).